R/utils-pipe.R

```diff
@@ -0,0 +1,11 @@
+#' Pipe operator
```

Comment:
I'm not sure what this is for, could you explain what the purpose of this

Reply:
This is just what's added by the `usethis::use_pipe()` function to enable pipe functionality throughout an R package. We can remove this since we are removing the pipe operator from the function.